pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -101,7 +124,7 @@ func TestGenCerts(_ *testing.T) {
 	apiserverCertCfg := NewCertConfig("karmada-apiserver", []string{""}, karmadaAltNames, &notAfter)
 	frontProxyClientCertCfg := NewCertConfig("front-proxy-client", []string{}, certutil.AltNames{}, &notAfter)
 
-	if err := GenCerts(TestCertsTmp, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
+	if err := GenCerts(TestCertsTmp, TestCaCertPath, TestCaKeyPath, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
 		fmt.Println(err)
```

Comment:
I found a problem with the previous code. The `TestGenCerts` function uses the `fmt. Println(err)` method to handle errors. In fact, this is meaningless. Can you help correct this error and change it to `t.Fatal(err)`? Just like:

```go
func TestGenCerts(t *testing.T) {
  ...
  if err := GenCerts(); err != nil {
    t.Fatal(err)
  }
  ...
}
```

Ps: here are more than one place has this error

Reply:
don't worried, let me create a issue for old code to tracking update `fmt.Println(err)`.

Reply:
https://github.com/karmada-io/karmada/pull/5175#discussion_r1675393622